src/fluree/db/server.clj

```diff
@@ -101,7 +101,7 @@
 (defn assoc-some
   "Assoc k -> v in m if v is not nil. Returns m unaltered otherwise."
   [m k v]
-  (if (nil? v)
+  (if-not (nil? v)
```

Comment:
D'oh! Good catch. I only did half of the logic inversion there... 🤦

Reply:
heh, yeah. i wish i had caught it when i reviewed the pr before though 😉